pmm-app/tests/DbaaS/verifyDBaaS_test.js

```diff
@@ -0,0 +1,242 @@
+    I.dontSeeElement(adminPage.fields.timePickerMenu);
+    I.click(dbaasPage.tabs.dbClusterTab.addDbClusterButton);
+    I.waitForElement(dbaasPage.tabs.dbClusterTab.monitoringWarningLocator, 30);
+    I.waitForText(dbaasPage.monitoringWarningMessage, 30);
+    await adminPage.verifyBackgroundColor(dbaasPage.tabs.dbClusterTab.optionsCountLocator(1), 'rgb(235, 123, 24)');
+    await adminPage.verifyBackgroundColor(dbaasPage.tabs.dbClusterTab.optionsCountLocator(2), 'rgb(142, 142, 142)');
+    I.click(dbaasPage.tabs.dbClusterTab.optionsCountLocator(2));
+    await adminPage.verifyBackgroundColor(dbaasPage.tabs.dbClusterTab.optionsCountLocator(1), 'rgb(224, 47, 68)');
+    await adminPage.verifyBackgroundColor(dbaasPage.tabs.dbClusterTab.optionsCountLocator(2), 'rgb(235, 123, 24)');
+    I.click(dbaasPage.tabs.dbClusterTab.optionsCountLocator(1));
+    await adminPage.verifyBackgroundColor(dbaasPage.tabs.dbClusterTab.optionsCountLocator(2), 'rgb(41, 156, 70)');
+    await adminPage.verifyBackgroundColor(dbaasPage.tabs.dbClusterTab.optionsCountLocator(1), 'rgb(235, 123, 24)');
+    I.click(dbaasPage.tabs.dbClusterTab.optionsCountLocator(2));
+    await dbaasAPI.apiUnregisterCluster(clusterName);
+  });
+
+Scenario('PMM-T456 Verify Create Cluster steps validation fields disabled/enabled @dbaas @not-pr-pipeline',
+  async (I, dbaasPage, dbaasAPI, adminPage) => {
+    if (!await dbaasAPI.apiCheckRegisteredClusterExist(clusterName)) {
+      await dbaasAPI.apiRegisterCluster(process.env.kubeconfig_minikube, clusterName);
+    }
+
+    I.amOnPage(dbaasPage.url);
+    dbaasPage.checkCluster(clusterName, false);
+    I.click(dbaasPage.tabs.dbClusterTab.dbClusterTab);
+    I.waitForElement(dbaasPage.tabs.dbClusterTab.addDbClusterButton, 30);
+    I.dontSeeElement(adminPage.fields.timePickerMenu);
+    I.click(dbaasPage.tabs.dbClusterTab.addDbClusterButton);
+    assert.strictEqual(
+      await I.grabAttributeFrom(dbaasPage.tabs.dbClusterTab.createClusterButton, 'disabled'),
+      true,
+      'Create Cluster Button Should be Disabled',
+    );
+    I.click(dbaasPage.tabs.dbClusterTab.optionsCountLocator(2));
+    assert.strictEqual(
+      await I.grabAttributeFrom(dbaasPage.tabs.dbClusterTab.createClusterButton, 'disabled'),
+      true,
+      'Create Cluster Button Should Still be Disabled',
+    );
+
+    assert.strictEqual(
+      await I.grabAttributeFrom(dbaasPage.tabs.dbClusterTab.advancedOptions.fields.memoryField, 'disabled'),
+      true,
+      'Memory Field Should be Disabled',
+    );
+    assert.strictEqual(
+      await I.grabAttributeFrom(dbaasPage.tabs.dbClusterTab.advancedOptions.fields.cpuNumberFields, 'disabled'),
+      true,
+      'Number of CPU Field Should be disabled',
+    );
+    assert.strictEqual(
+      await I.grabAttributeFrom(dbaasPage.tabs.dbClusterTab.advancedOptions.fields.diskSizeInputField, 'disabled'),
+      true,
+      'Disk Size field must be disabled',
+    );
```

Comment:
```suggestion
    assert.ok(
      await I.grabAttributeFrom(dbaasPage.tabs.dbClusterTab.advancedOptions.fields.diskSizeInputField, 'disabled'),
      'Disk Size field must be disabled',
    );
```
I think this way it should also work.

Reply:
sure, makes sense I will update

Reply:
Done